src/main/java/com/questhelper/quests/gettingahead/GettingAhead.java

```diff
@@ -293,6 +293,7 @@ public void setupSteps()
 		takeSaw = new DetailedQuestStep(this, new WorldPoint(1239, 3696, 0), "Take the saw in the building north of the farmhouse.", saw);
 		takeHammer = new DetailedQuestStep(this, new WorldPoint(1259, 3686, 0), "Take the hammer east of the farmhouse.", hammer);
 		takeNails = new ObjectStep(this, ObjectID.WORKBENCH_40364, new WorldPoint(1239, 3699, 0), "Take some nails from the farm's workbench.");
+		takeNails.addDialogStep("Take the nails.");
```

Comment:
Not clear enough. Can we do a pop up on the user's pc?

Reply:
I will give you a flashing notification when clicking the workbench

Reply:
K. Good enough for me!